src/main/java/com/hivemq/extensions/iteration/AsyncLocalChunkIterator.java

```diff
@@ -32,10 +32,10 @@
 /**
  * @author Christoph Schäbel
  */
-public class AsyncLocalChunkIterator<K, V> implements AsyncIterator<K, V> {
+public class AsyncLocalChunkIterator< V> implements AsyncIterator< V> {
```

Comment:
please remove the whitespaces at all < V>

Reply:
👍